.github/workflows/swift-toolchain.yml

```diff
@@ -906,19 +904,29 @@ jobs:
 
       - uses: compnerd/gha-setup-vsdevenv@f1ba60d553a3216ce1b89abe0201213536bc7557 # main
         with:
-          host_arch: amd64
+          host_arch: ${{ inputs.build_arch }}
           components: 'Microsoft.VisualStudio.Component.VC.Tools.x86.x64;Microsoft.VisualStudio.Component.VC.Tools.ARM64'
           arch: ${{ matrix.arch }}
 
       - name: Install Swift Toolchain
+        if: inputs.build_arch == 'amd64'
         uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
         with:
           github-repo: thebrowsercompany/swift-build
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          release-asset-name: installer-amd64.exe
+          release-asset-name: installer-${{ inputs.build_arch }}.exe
           release-tag-name: '20231016.5'
+      - name: Install Swift Toolchain
+        if: inputs.build_arch == 'arm64'
+        uses: compnerd/gha-setup-swift@b6c5fc1ed5b5439ada8e7661985acb09ad8c3ba2 # main
+        with:
+          github-repo: thebrowsercompany/swift-build
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          release-asset-name: installer-${{ inputs.build_arch }}.exe
+          release-tag-name: '20231016.0'
 
       - uses: nttld/setup-ndk@v1
+        if: inputs.build_os != 'Windows' || inputs.build_arch != 'arm64'
```

Comment:
Why can this not run on Windows ARM64?

Reply:
Because there is no NDK for Windows ARM64